build/preinstall.sh.cmake

```diff
@@ -29,7 +29,7 @@ mecho()
 
 ###############################################################################
 
-COMPILEFOR=@COMPILEFOR@
+SYSTEM_NAME=@CMAKE_SYSTEM_NAME@
```

Comment:
Because I wanted to avoid CMAKE_ prefixes in shell scripts. 1) Because they can cause problems when INSTALL is not used carefully, and 2) because it has nothing to do with CMake anymore and COULD come from other sources. "SYSTEM_NAME" makes it still clear, what it is and where it came from